### ISSUE TITLE
Support for argument intelligence?

### ISSUE THREAD

**Issue by rwatts3** [atom/ide-typescript#9]
Greetings, 

Will there be support or is there a roadmap for showing suggestions of arguments within a function?

```js
var foo = function(hello);

foo() // would prompt the user for "hello : any" when the user begins typing foo.(
```

**Comment by damieng** [comment]
Hmm, not exactly sure how we'd do this as there's nothing in the spec for it... but it's possible maybe that we could take the entire arg signature, strip the types and turn it into a snippet. Something like that  would be after the main release though.

The other option is it would be great to have an on-screen decoration show you the parameter names for the method you're writing. Doesn't stop you having to type as much but does at least let you see all the args while you edit.

**Comment by damieng** [comment]
Shipped in v0.7.0 !
